FestivalPlanner/loader/forms/loader_forms.py

```diff
@@ -411,9 +413,10 @@ def pop_key_fields(self, value_by_field):
 
 class FilmLoader(SimpleLoader):
     expected_header = ['seqnr', 'filmid', 'sort', 'title', 'titlelanguage',
-                       'section', 'duration', 'mediumcategory', 'url']
+                       'section', 'duration', 'mediumcategory', 'reviewer', 'url']
```

Comment:
Please consider to define this in the config file and use it both here and in the loader (`planner_interface.py`).

Reply:
Done.
Plus fixed the spelling warnings (e.g. `filmid` ->`film_id`).
Plus corrected all existing headers as to be able to still load the files. 😡 🥲 😎